waiter/project.clj

```diff
@@ -88,6 +90,7 @@
                  ;; use maven to download this jar so we can set up the boot classpath
                  [org.mortbay.jetty.alpn/alpn-boot "8.1.13.v20181017"
                   :scope "provided"]
+                 [org.opensaml/opensaml "2.6.4"]
```

Comment:
Have you confirmed using `lein deps` that we don't have any dependency conflicts by introducing this? If so, we should resolve those conflicts too.

Reply:
lein deps runs with no error

Reply:
Can we try `lein deps :tree`